docs/dependencies.md

```diff
@@ -19,8 +19,8 @@ Add the library based on the name listed in [Maven Central](http://search.maven.
 To update a library's version, simply change the version number declared in `build.gradle` file.
 
 - If you are using command line, your next build will automatically reflect the change.
-- If you are using Eclipse, right click on the project in the Project Explorer and select `Gradle → Refresh Gradle Project` for the changes to be reflected.
```

Comment:
Managing dependencies is not just IDE-related, the way you are doing this right now is to let the developers know how to update dependencies but treating it as black-box unless they somehow visit this document or already know how dependencies are managed.